enterprise/management/commands/assign_enterprise_user_roles.py

```diff
@@ -87,9 +108,27 @@ def _get_enterprise_customer_users_batch(self, start, end):
         Returns a batched queryset of EnterpriseCustomerUser objects.
         """
         LOGGER.info('Fetching new batch of enterprise customer users from indexes: %s to %s', start, end)
-        return User.objects.filter(pk__in=EnterpriseCustomerUser.objects.values('user_id'))[start:end]
+        return User.objects.filter(pk__in=self._get_enterprise_customer_user_ids())[start:end]
 
-    def _assign_enterprise_role_to_users(self, _get_batch_method, options):
+    def _get_enterprise_enrollment_api_admin_users_batch(self, start, end):     # pylint: disable=invalid-name
```

Comment:
This also should be converted to a static method.

Reply:
I guess this isn't called outside this management command, and I don't see any other code calling this in near future and also may be this command would be deleted ... so I am leaving it as is.